kumascript/src/api/page.js

```diff
@@ -56,7 +56,17 @@ module.exports = {
   // the results
   //
   subpagesExpand(path, depth, self) {
-    return this.info.getChildren(path || this.env.url, self);
+    try {
+      return this.info.getChildren(path || this.env.url, self);
+    } catch (error) {
+      this.env.recordNonFatalError(
+        "bad-pages",
```

Comment:
I'm a bit at loss with what to call this. Truth be told, I could call this anything here because in `kumascript/src/render.js` it "converts" this identifier to a class and that class's name is what appears in the flaw when it "travels over to yari".